hyper/hyper/suspend_test.go

```diff
@@ -0,0 +1,58 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"libvirt.org/libvirt-go"
+)
+
+func TestSuspend(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) *libvirt.Domain
+		ExpectedErr    string
+	}{
+		"suspend the desktop correctly": {
```

Comment:
Should check the desktop state afterwards

Reply:
ok, done

Reply:
we should rely on libvirt functions.